extensions/sftp/tests/PutSFTPTests.cpp

```diff
@@ -244,7 +240,7 @@ TEST_CASE_METHOD(PutSFTPTestsFixture, "PutSFTP bad password", "[PutSFTP][authent
 }
 
 TEST_CASE_METHOD(PutSFTPTestsFixture, "PutSFTP public key authentication success", "[PutSFTP][authentication]") {
-  plan->setProperty(put, "Private Key Path", utils::file::concat_path(get_sftp_test_dir(), "resources/id_rsa"));
+  plan->setProperty(put, "Private Key Path", get_sftp_test_dir() / "resources/id_rsa");
```

Comment:
I've changed these and the ones in ListSFTPTests in https://github.com/apache/nifi-minifi-cpp/pull/1424/commits/b7fce3d7cf85180b6054df874a0ad7b5b6f2e31e
I havent changed the occurrences where the root is a string aswell (e.g. ["nifi_test/tstFile.ext"](https://github.com/martinzink/nifi-minifi-cpp/blob/MINIFICPP-1862-2/extensions/sftp/tests/ListSFTPTests.cpp#L209), because I think that could hinder the readability and std::filesystem is capable to correctly parse those forward slashes even on windows, but I am curious about your opinion.